packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -169,7 +169,7 @@ class SnackBar extends StatelessWidget {
 
   /// The amount of time the snack bar should be displayed.
   ///
-  /// Defaults to 1.5s.
+  /// Defaults to 4.0s.
```

Comment:
The material spec link above this line isn't valid (anymore). It should be https://material.io/design/components/snackbars.html#behavior

Reply:
You want me to copy in that whole "Snackbars appear without warning, and don't require user interaction. They automatically disappear from the screen after a minimum of four seconds, and a maximum of ten seconds."?

Reply:
I was just suggesting that you replace the URL on line 179 (`https://material.google.com/components/snackbars-toasts.html`)

Reply:
Just changed that!